.github/workflows/deploy-dev.yml

```diff
@@ -103,7 +103,7 @@ jobs:
           IMAGE_TAG: dev-ckan-${{ github.sha }}
         with:
           command: >-
-            helm upgrade ckan -n dev --repo https://keitaro-charts.storage.googleapis.com --install --wait ckan -f ./charts/ckan/values.yaml
+            helm upgrade ckan -n dev --repo https://marvell-consulting.github.io/ckan-helm-chart --install --wait ckan -f ./charts/ckan/values.yaml
```

Comment:
Is there meant to be a file at https://marvell-consulting.github.io/ckan-helm-chart/charts/ckan/values.yaml ?

Context: there is not, but there is one at https://marvell-consulting.github.io/ckan-helm-chart/charts/values.yaml

Reply:
My best guess is that the github pages branch needs syncing to include [this commit](https://github.com/Marvell-Consulting/ckan-helm-chart/commit/57e1ebc0da7af62a0572498a65230b8baa1749b6)

Reply:
Maybe. It's not entirely clear to me how the [publisher action](https://github.com/marketplace/actions/helm-chart-releaser) expects us to structure the directory. I followed the discussion in https://github.com/helm/chart-releaser-action/issues/36 and it seemed like nesting in `charts` was the expected behaviour but we can prod this